submit_coverity_scan.sh

```diff
@@ -99,9 +101,9 @@ quietfilter()
 }
 
 if [ -n "$GITURL" ]; then
-	TEMPDIR=`mktemp -d /tmp/cov-scan-XXXXXX`
-	git clone $QUIETOPT "$GITURL" $TEMPDIR
-	cd $TEMPDIR
+	TEMPDIR=$(mktemp -d /tmp/cov-scan-XXXXXX)
+	git clone $QUIETOPT "$GITURL" "$TEMPDIR"
```

Comment:
I assume you silenced $QUIETOPT somewhere above?

Reply:
No, so I assume shellcheck is smart enough to realize it only ever contains strings that don't need quoting.